src/layout.js

```diff
@@ -444,14 +445,15 @@ export function applyStaticLayout(element) {
   } else if (layout == Layout.INTRINSIC) {
     // Intrinsic uses an svg inside the sizer element rather than the padding trick
     // Note a naked svg won't work becasue other thing expect the i-amphtml-sizer element
-    const sizer = element.ownerDocument.createElement('i-amphtml-sizer');
-    const intrinsicSizer = element.ownerDocument.createElement('img');
-    sizer.classList.add('i-amphtml-sizer');
-    intrinsicSizer.classList.add('i-amphtml-intrinsic-sizer');
+    const sizer = htmlFor(element)`
+      <i-amphtml-sizer class="i-amphtml-sizer">
+        <img class="i-amphtml-intrinsic-sizer" />
+      </i-amphtml-sizer>`;
+    const intrinsicSizer = sizer.firstElementChild;
     intrinsicSizer.setAttribute('src',
```

Comment:
Perhaps I am missing something. Why can't this attribute be included in the template literal?